block.go

```diff
@@ -0,0 +1,42 @@
+package slack
+
+// @NOTE: Blocks are in beta and subject to change.
+
+// More Information: https://api.slack.com/block-kit
+
+var (
```

Comment:
these should be constants. with a named string type. removes the need for all the `ValidateBlock` methods

can look something like this.

```golang
type MessageBlockType string

const (
  mbtSection MessageBlockType = "section"
  mbtDivider = "divider"
  mbtImage = "image"
  // ...
)
```